### DISCUSSION TITLE
ImmoScout 24 with "Plus+" membership

### DISCUSSION THREAD

**Discussion by benjibee** [orangecoding/fredy#90] [Ideas]
Finally got this up and running on an EC2 instance, and will be sharing it with some friends and trying to maintain it for them. Signing up for a premium account from ScrapingAnt, too.

We're mostly in Berlin and ImmoScout is sadly the only real option for most of us. They have a tricky setup because they provide a premium "Plus+" account that may see some listing before others. I'm looking into setting that up perhaps with setting cookies, etc.

The question is: is this on your radar at all? Is there a need for it from others? I'm not a JavaScript developer so I'm not sure I could get a useful PR together, but I'm looking into it.

**Comment by orangecoding** [comment]
Hey.

I know they have kind of a unique selling point by holding a monopoly onto real estate market in Germany, but honestly I haven't looked into it as I don't rly like this style of making business.

Furthermore, scraping is somewhat accepted, but I'm pretty sure that they have something in their terms that denies using your account to do automagic screening.

AFAIK if you're willing to pay that money, they give you a pretty good email alerting function..?

**Reply by benjibee** [reply]
I wish they did have a good notification system, but they don't. Flats are often posted and taken down within an hour or two (because of the high volume of inquiries) and sometimes the notification comes an hour or two after the ad goes up. Not a great combo. Anyway I don't see lots of ads going up for the upgraded membership, just thought I would ask about the cookies thing in case it was a need elsewhere.
